tests/baselines/reference/privateNameFieldDestructuredBinding.js

```diff
@@ -0,0 +1,62 @@
+//// [privateNameFieldDestructuredBinding.ts]

```

Comment:
I think this is left over from a previous test run as using the vary-by-configuration test settings for `// @target:` should not produce a file without a `(target=x)` suffix.